tests/unit/result_handlers/test_text.py

```diff
@@ -39,7 +39,7 @@ def test_import(module_under_test):
 def test_basic_result_handler(module_under_test):
     """Test basic handler executes """
     result_df = DataFrame(SAMPLE_RESULT_DATA)
-    result_handler = module_under_test.TextResultHandler()
+    result_handler = module_under_test.TextResultHandler(format="table")
```

Comment:
Can you add a unit test (maybe in unit/test_cli_tools.py) that checks if a user passes in an invalid format an error will be thrown?

Reply:
Done.